tsdb/index/index.go

```diff
@@ -647,9 +786,10 @@ func (w *Writer) writePostings() error {
 		}
 		offsets = append(offsets, uint32(startPos/16))
 		// Skip to next series.
-		d.Skip(d.Uvarint() + crc32.Size)
+		x := d.Uvarint()
+		d.Skip(x + crc32.Size)
 		if err := d.Err(); err != nil {
-			return nil
```

Comment:
:scream:

Reply:
The good news is it only happened when you're in the middle of debugging a broken rewrite of an index writer :)